flake-modules/dev/default.nix

```diff
@@ -24,6 +24,23 @@
         };
       };
 
+      apps.generate-files.program = pkgs.writeShellScriptBin "generate-files" ''
+        set -e
+
+        repo_root=$(git rev-parse --show-toplevel)
+        generated_dir="$repo_root/generated"
+
+        # Rust-Analyzer
+        nix build .\#rust-analyzer-options
+        cat ./result >"$generated_dir"/rust-analyzer.nix
+
+        nix fmt
+      '';
+
+      packages = {
+        rust-analyzer-options = pkgs.callPackage ./updates/rust-analyzer.nix { };
+      };
```

Comment:
This can be in `apps` too, seeing as it's really something to _run_ not _build_.

Reply:
Nevermind. It is just a derivation, not a derivation that produces a script so `packages` is correct.

Assuming it needs to be a flake output at all (for debugging you could just callPackage it with nix-build?

Reply:
I find it easier to debug by using it as a flake output, because
- I never remember the right invocation of nix-build
- Rust-analyzer from our flake inputs will be used